node/querystring.ts

```diff
@@ -2,7 +2,114 @@
 import { Buffer } from "./buffer.ts";
 import { ERR_INVALID_URI } from "./_errors.ts";
 
-export interface ParsedUrlQuery {
+/******************** querystring.decode() ********************/
```

Comment:
I'm afraid this kind of comments are just confusing to other contributors because we don't have rule to group functions & variable by the position in the source code

Reply:
I reffered https://github.com/denoland/deno_std/blob/905408229b64a4b8b1b3886135fb7f1dae13417d/testing/fast_check_example.ts but this is just example so I deleted. https://github.com/denoland/deno_std/pull/1531/commits/7d04d5fd179fc24a4056b9a0515639a6ad2d52a9